src/KubernetesPfSenseController/Plugin/DNSResourceTrait.php

```diff
@@ -69,10 +69,14 @@ public function doAction()
 
             if ($dnsmasqEnabled) {
                 $dnsmasqConfig = PfSenseConfigBlock::getRootConfigBlock($this->getController()->getRegistryItem('pfSenseClient'), 'dnsmasq');
+                if (!isset($dnsmasqConfig->data) || !is_array($dnsmasqConfig->data)) {
+                    $dnsmasqConfig->data = [];
+                }
```

Comment:
There was a bug here if the hosts config is empty, it returns an empty string; so fixed that here and on unbound